astropy/time/formats.py

```diff
@@ -690,10 +690,19 @@ def _default_scale(self):
 
 class TimeUnix(TimeFromEpoch):
     """
-    Unix time: seconds from 1970-01-01 00:00:00 UTC, ignoring leap seconds.
+    Unix time (UTC): seconds from 1970-01-01 00:00:00 UTC, subject to leap seconds.
 
     For example, 946684800.0 in Unix time is midnight on January 1, 2000.
 
+    In unix time, each UTC day always has exactly 86400 seconds, even on
```

Comment:
Please keep PR's in scope.  I think that the new content is actually confusing the situation because of the mix here between POSIX unix time and astropy `unix_time`. In particular with the new text, the meaning of "NOTE: this quantity .." becomes ambiguous. I think the original paragraph is sufficient, so please just revert these new paragraphs.